src/utils.js

```diff
@@ -407,15 +408,14 @@ module.exports = (function() {
       var flickrURL = url + "?" + queryString + signature;
 
       var req = request.post(flickrURL, function(error, response, body) {
-        // format:json does not actually work, so we need to grab the photo ID from the response XML:
-        // <?xml version="1.0" encoding="utf-8" ?>\n<rsp stat="ok">\n<photoid>.........</photoid>\n</rsp>\n
-        var data;
         if(!body) {
-          error = error || "No body found in response";
-        } else if (body.indexOf('rsp stat="ok"')>-1) {
-          data = parseInt(body.split("<photoid>")[1].split("</photoid>")[0], 10);
-        }
-        callback(error, data);
+          return callback("No body found in response");
```

Comment:
can this still be updated so that it's consistent with the other callback invocations?
```js
return callback("No body found in response", null);
```